lib/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ mark_as_advanced(VGG_AUTOMERGE_INCLUDE)
 option(open_vgg_layout_test "Disable vgg_layout test" OFF)
 add_subdirectory(vgg_contrib/yoga EXCLUDE_FROM_ALL)
 add_subdirectory(vgg_layout EXCLUDE_FROM_ALL)
+if(EMSCRIPTEN)
+  target_compile_definitions(flexbox PUBLIC BOOST_NO_CXX98_FUNCTION_BASE)
```

Comment:
I don't think it is good practice to change target properties outside of its own project.